In [3]:
import numpy as np 

import pandas as pd 

import seaborn as sns
%matplotlib inline
from matplotlib import pyplot as plt
from matplotlib import style

In [4]:
df = pd.read_excel('Data_Pathrise.xlsx')


In [11]:
df.head(2)

id pathrise_status primary_track cohort_tag  program_duration_days  placed  \
0   1          Active           SWE     OCT19A                    NaN       0   
1   2          Active           PSO     JAN20A                    NaN       0   

  employment_status  highest_level_of_education length_of_job_search  \
0         Unemployed          Bachelor's Degree           3-5 months   
1         Unemployed    Some College, No Degree           3-5 months   

           biggest_challenge_in_search professional_experience  \
0      Hearing back on my applications               3-4 years   
1  Getting past final round interviews               1-2 years   

  work_authorization_status  number_of_interviews  number_of_applications  \
0            Canada Citizen                   2.0                     900   
1                   Citizen                   6.0                       0   

  gender                                 race  
0   Male  Non-Hispanic White or Euro-American  
1   Male  Non-Hispanic White or Euro-American

In [9]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2544 entries, 0 to 2543
Data columns (total 16 columns):
 #   Column                       Non-Null Count  Dtype  
---  ------                       --------------  -----  
 0   id                           2544 non-null   int64  
 1   pathrise_status              2544 non-null   object 
 2   primary_track                2544 non-null   object 
 3   cohort_tag                   2536 non-null   object 
 4   program_duration_days        1928 non-null   float64
 5   placed                       2544 non-null   int64  
 6   employment_status            2315 non-null   object 
 7   highest_level_of_education   2486 non-null   object 
 8   length_of_job_search         2470 non-null   object 
 9   biggest_challenge_in_search  2520 non-null   object 
 10  professional_experience      2322 non-null   object 
 11  work_authorization_status    2260 non-null   object 
 12  number_of_interviews         2326 non-null   float64
 13  number_of_applicat

In [12]:
df.describe()

id  program_duration_days       placed  number_of_interviews  \
count  2544.000000            1928.000000  2544.000000           2326.000000   
mean   1272.500000             136.098548     0.375786              2.182287   
std     734.533866             125.860248     0.484420              2.959273   
min       1.000000               0.000000     0.000000              0.000000   
25%     636.750000              14.000000     0.000000              0.000000   
50%    1272.500000             112.000000     0.000000              1.000000   
75%    1908.250000             224.000000     1.000000              3.000000   
max    2544.000000             548.000000     1.000000             20.000000   

       number_of_applications  
count             2544.000000  
mean                36.500786  
std                 53.654896  
min                  0.000000  
25%                  9.000000  
50%                 20.000000  
75%                 45.000000  
max               1000.000000

The Dataset has 2544 examples and 14 features + the target variable (placed). 
2 of the features are floats, 3 are integers and 11 are objects.

### Features short description
**id** = unique id of a fellow </br>
**pathrise_status** = status of a fellow in the program: *Active .. Closed .. Withdrawn ... etc* </br>
**primary_track** = field of track: *Data .. Design .. Web .. etc* </br>
**cohort_tag** = Cohort tag: *monthname|year|(A/B) for start on first/third week of month* </br>
**program_duration_days** = how many days a fellow was in the program, N/A for current fellows </br>
**placed** = whether a fellow was placed while going through the program (pathrise_status==placed) </br>
**employment_status** = employment status when joined the program: *Employed .. copntractor .. etc* </br>
**highest_level_of_education** = highest level of education when joined the program   </br>
**length_of_job_search** = length of a job search before joining the program  </br>
**biggest_challenge_in_search** = biggest challenge in a job search (self-reported) </br>
**professional_experience** = professional work exeperience before joining the program  </br>
**work_authorization_status** = work authorization status before joining the program </br>
**number_of_interviews** = number of interviews before joining the program (self-reported) </br>
**number_of_applications** = number of applications before joining the program (self-reported) </br>
**gender** = gender </br>
**race** = race </br>
